styles/style.css

```diff
@@ -0,0 +1,231 @@
+html{
+    overflow-x: hidden;
+    background-color: #00b0f025;
+    font-family: 'Open Sans Condensed', sans-serif;
+}
+body{
+    margin: 0;
+    padding-top: 10vh;
+}
+
+
+
+/*----header-----*/
+
+#header{
+    text-align: center;
+}
+
+#OF{
+    height: 15vh;
+    z-index: 30;
+    padding-bottom: 0;
+}
+
+#header hr{
+    width: 50%;
+    height: 7px;
+}
+
+#features-title{
+    text-align: center;
+    margin-top: 7vh;
+}
+
+#f-button-wrap{
+    display: flex;
+    justify-content: center;
+    margin-top: 3vh;
+    margin-bottom: 3vh;
+}
+
+
+#features{
+    display: flex;
+    justify-content:space-around;
+    margin-left: 20vw;
+    margin-right: 20vw;
+    margin-bottom: 7vh;
+}
+figure{
+    margin:0;
+}
+
+figcaption{
+    text-align: center;
+    margin-top: 7vh;
+}
+
+.features-ph{
+    height: 24vh;
+    width: 17vw;
+    border: 3px inset lightgray;
+}
+
+/*----featured images----*/
+#f1{
+    background: url(https://cdn.thewirecutter.com/wp-content/uploads/2018/07/roomfans-2x1-9853.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    
+}
+
+#f2{
+    background: url(/projects/remock-d-morg/Images/fans/bedfan.jpg); 
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    
+}
```

Comment:
So initially when I started up your webpage I noticed some of your images were broken. Upon investigating, I noticed a couple things. This first is that you have brought in all of your images through your CSS styling as `background:` images as opposed to adding `href="image url or path here"` within your image tags on the HTML document which is fine. However the reason they are not working is just due to the path the program is looking down for the images. 

To try and create a visual for you . . from within this images folder we are telling the program to look inside this folder for another folder named /projects -> then inside a folder named /remock-d-morg -> then inside /Images -> for a file named /bedfan.jpg

When the correct path from this styling folder would actually be to go out one level using `..` -> then to go into the images folder /Images -> then into the /fans folder -> and finally to find an image named /bedfan.jpg. 

So all together the path you want on line 75 would look like this `../Images/fans/bedfan.jpg`
../Images/fans/bedfan.jpg

Reply:
Very well, I'm curious why the paths were working fine on my end? Is that due to them being local on my vsCode?

Reply:
Are you launching the program from withing the specific program file? Or possible from outside that programs file in VS code